documentation/prerequisites.md

```diff
@@ -4,24 +4,40 @@ title: Prerequisites
 
 Following is the list of prerequisites for running Kubernetes with vSphere Cloud Provider:
 
-* Kubernetes version - 1.6.5 onwards. Tested with v1.6.5, v1.7, v1.8. 
-* vSphere version - 6.0.x onwards.
-* Container Host operating systems - 
-    - Photon - v1.0 GA 
-    - Ubuntu - v16.04 
-    - CoreOs - v4.11.2 
+* Kubernetes version - v1.6.5 and above
+
+Following table summarizes key features introduced in vSphere Cloud Provider in each Kubernetes release
+
+| Kubernetes Release | vSphere Cloud Provider feature |
+| ------ | ------ |
+| v1.6.3 | Dynamic volume provisioning using vSAN storage capabilities |
+| v1.6.5 | Integration with vSphere HA |
+| v1.7.0 | Integration with vSphere Storage Policy Based Management(SPBM) for dynamic volume provisioning |
+| v1.7.0 | Enhanced vSphere Cloud Provider debuggabilty via integration with metrices exposed for Kubernetes storage APIs |
+| v1.8.0 | vSphere Cloud Provider refactoring for better debuggability, logging and code maintenance |
+| v1.8.2 | Performance improvement for large scale deployment |
+| v1.9.0 | Multi vCenter Support |
+
+* vSphere version - 6.0.x and above
+
+* Container Host operating systems -
+    - Photon - v1.0 GA
+    - Ubuntu - v16.04
+    - CoreOs - v4.11.2
     - Centos  - v7.3
     - RHEL OS -  v7.4
+
 * vSphere setup to deploy the Kubernetes cluster.
-* vSphere shared storage. It can be any one of VSAN, sharedVMFS, sharedNFS.
-   - Shared storage allows all the nodes in the Kubernetes cluster to access the storage blocks.
+   - For kubernetes version 1.9.x and above: vSphere Cloud Provider supports kubernetes cluster spanning across multiple vCenters.
+   - For kubernetes version 1.8.x and below: vSphere Cloud Provider supports kubernetes cluster deployed only in one vCenter.
+* vSphere supported storage.
+   - It can be HCI offering such as VMware vSAN or block and file storage offerings like VMFS and NFS.
```

Comment:
Formatting for this and line above doesn't appear correct.

Reply:
fixed.